pkg/csi_driver/node.go

```diff
@@ -68,8 +79,15 @@ func (s *nodeServer) NodePublishVolume(ctx context.Context, req *csi.NodePublish
 	// Check if mount already exists
 	mounted, err := s.isDirMounted(targetPath)
 	if err != nil {
-		return nil, err
+		if goOs == "windows" {
+			if !os.IsNotExist(err) {
```

Comment:
Add a comment here, something like "on Windows the mount target must not exist yet".

Also, you can combine with previous line: goOs == "windows"  and !os.IsNotExist(err)

Reply:
In the current code, if goOs is Windows and the error is IsNotExist, the code won't do anything. Example:
```go
if goOs == "windows" {
    if !os.IsNotExist(err) {
        return nil, err
    }
    // if goOs == "windows" and os.IsNotExist(err)
    //do nothing
} else {
    return nil, err
}
```

If L82 and L83 are combined then this is lost.

Reply:
Oops, you're right. Looks good as it is.